src/geometry/manifold/mfderiv.lean

```diff
@@ -1232,6 +1232,70 @@ lemma mfderiv_within_const (hxs : unique_mdiff_within_at I s x) :
 
 end const
 
+section arithmetic
+/-! #### Arithmetic -/
+
+variables {S: topological_space.opens 𝕜}
+
+lemma mdifferentiable_add {f g : S → 𝕜} (hf : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) f)
+  (hg : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) g) : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) (f + g) :=
+begin
+  simp_rw mdifferentiable at *,
+  simp only [mdifferentiable_at, differentiable_within_at_univ] at *,
+  intro x,
+  split,
+  { apply continuous_at.add (hf x).1 (hg x).1 },
+  { convert (differentiable_within_at.add (hf x).2 (hg x).2) },
+end
+
+lemma mdifferentiable_mul {f g : S → 𝕜} (hf : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) f)
+  (hg : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) g) : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) (f * g) :=
+begin
+  simp_rw mdifferentiable at *,
+  simp only [mdifferentiable_at, differentiable_within_at_univ] at *,
+  intro x,
+  split,
+  apply continuous_at.mul (hf x).1 (hg x).1,
+  convert (differentiable_within_at.mul (hf x).2 (hg x).2),
+end
+
+lemma mdifferentiable_smul {f : S → 𝕜} (s : 𝕜) (hf : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) f) :
+  mdifferentiable 𝓘(𝕜) 𝓘(𝕜) (s • f) :=
+begin
+  simp_rw mdifferentiable at *,
+  simp only [mdifferentiable_at, differentiable_within_at_univ] at *,
+  intro x,
+  split,
+  { apply continuous_at.const_smul (hf x).1,
+  exact has_continuous_smul.has_continuous_const_smul },
+  apply differentiable_within_at.congr (differentiable_within_at.const_smul (hf x).2 s),
+  { intro x, simp },
+  { refl },
+end
+
+lemma mdifferentiable_zero : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) (0 : S → 𝕜) :=
+begin
```

Comment:
Already exists as [mdifferentiable_at_const](https://leanprover-community.github.io/mathlib_docs/geometry/manifold/mfderiv.html#mdifferentiable_at_const)

Reply:
Same for `mdifferentiable_one`